### ISSUE TITLE
[Bug Report][3.5.16] Indeterminate checkbox aria-checked doesn't match control value

### ISSUE THREAD

**Issue by justingish** [vuetifyjs/vuetify#19647]
### Environment
**Vuetify Version:** 3.5.16
**Vue Version:** 3.4.23
**Browsers:** Chrome 124.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Inspect the checkbox element

### Expected Behavior
Since it is using a native HTML checkbox control, no aria-checked attribute should be present. 

### Actual Behavior
The checkbox control has aria-checked set to 'mixed' which doesn't match the native control's value.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqLrlYqLkrWdywo0CsrTVWyUrIpSc0tyEksSbWLyVNQsCnTTc7PK0nMzEstUkjLKc1MAQtDJDJSk7OT8isUMvNSUktSi3Iz84DaFPQhGvWRdAJFbPSRzVWq1ckrzckBE7EAobEtow==)

### Other comments
[MDN article](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role) about aria checkbox role.  

When using a native html checkbox, you should not need the role or aria-checked attributes. Because the VCheckbox does use the aria-checked attribute, it causes a mismatch between the control's value and the aria-checked value.  

I suggest simply removing aria-checked from the control.

**Comment by johnleider** [comment]
We currently implement aria-checked in all of these components. Does the same apply here as well?

![Image](https://github.com/user-attachments/assets/c67632ea-f269-4c51-9b91-5f1897bb36fe)
